app/javascript/dashboard/store/modules/helpCenterPortals/actions.js

```diff
@@ -0,0 +1,90 @@
+import HelpCenterPortalsAPI from 'dashboard/api/helpCenter/portals.js';
+
+export const actions = {
+  fetchAllHelpCenters: async ({ commit }) => {
```

Comment:
Let us follow the naming convention here. https://discord.com/channels/897869226579222540/999557432747503698/999557936294678539